src/app/integrations/list/list.component.html

```diff
@@ -106,27 +106,28 @@ <h4 class="modal-title pull-left">Confirm {{ getActionTitle() }}</h4>
               </h3>
             </div>
 
-            <div *ngIf="integration.currentStatus !== 'Pending'">
-              <span *ngIf="integration.currentStatus === 'Activated'"
-                    class="label label-primary"
-                    (click)="requestDeactivate(integration)">Active</span>
-              <span *ngIf="integration.currentStatus === 'Deactivated'"
-                    class="label label-default"
-                    (click)="requestActivate(integration)">Inactive</span>
-              <span *ngIf="integration.currentStatus === 'Draft'"
-                    class="label label-warning"
-                    disabled>Draft</span>
-              <span *ngIf="integration.currentStatus === 'Deleted'"
-                    class="label label-danger"
-                    disabled>Deleted</span>
+            <div *ngIf="integration.currentStatus !== 'Pending' && integration.currentStatus !== 'Draft'">
+              <label class="switch_wrap"
+                     for="field-{{ integration.id }}">
+                <input id="field-{{ integration.id }}"
+                       type="checkbox"
+                       value="{{ integration.currentStatus === 'Activated' ? 0 : 1 }}" />
+                <div class="switch"><span class="bullet"></span></div>
+              </label>
+            </div>
+
+            <div *ngIf="integration.currentStatus === 'Draft'"
+                 class="draft">
+              <span class="label label-warning" disabled>Draft</span>
             </div>
           </div>
         </div>
 
 
         <!-- Metrics & Actions -->
 
-        <div class="col-xs-7">
+        <div class="col-xs-7"
+             (click)="goto(integration)">
```

Comment:
Leaving it for the rest of the sub-divisions though like the `uses` etc. I forgot the kabob was in that same group. :/